### DISCUSSION TITLE
Has anyone seen the FC config being spontaneously reset to defaults ?

### DISCUSSION THREAD

**Discussion by mirov** [iNavFlight/inav#6410] [Q&A] [answered]
I've got a FLYWOOF411 that has twice now been working just fine, but suddenly when powered on boots up in the all defaults state (ie: like what you get if you erase flash and update the firmware).  I've had CLI backups, but this is troubling.  I'm using 2.6.0 with a customized target for a fixed wing.  Is this something that just happens now and then, or is there something that I might be doing to cause it ?

-Russ

**Comment by digitalentity** [comment, marked as answer]
I've never seen this happening. Looks like a hardware issue to me
